src/main/java/com/svydovets/bibirnate/session/impl/SessionFactoryImpl.java

```diff
@@ -13,6 +15,10 @@ public class SessionFactoryImpl implements SessionFactory {
 
     @Override
     public Session openSession() {
-        return new SessionImpl(dataSource);
+        try {
+            return new SessionImpl(dataSource.getConnection());
+        } catch (SQLException ex) {
+            throw new JdbcException(ex.getMessage(), ex);
```

Comment:
need a normal description here, like: Cannot open session. The purpose is [ex.getMessage]